### PR TITLE
Enable SQ & update conformance

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Rollback #1945 
- Update conformance test parameters & golds

### Reason for changes

- Need to have enabled SQ

### CODE REVIEW

**Comment by KodiaqQ** [comment]
run pylint pre-commit tests
run pytorch pre-commit tests
run tensorflow pre-commit tests
run openvino pre-commit tests
run onnx pre-commit tests

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: tests/post_training/reference_data.yaml [hunk 1]
```diff
@@ -62,7 +62,7 @@ timm/deit3_small_patch16_224_backend_ONNX:
     metric_value: 0.76806
     metric_value_fp32: 0.76974
 timm/deit3_small_patch16_224_backend_OV:
-    metric_value: 0.76786
+    metric_value: 0.76582
```
[the remark this answers is not in the document]
Comment (KodiaqQ) [reply tests/post_training/reference_data.yaml]: Done

**Comment by KodiaqQ** [comment]
Conformance re-run - 114

**Comment by l-bat** [comment]
It would be great to add progress bar for SQ algorithm, as it is done for BC/FBC.
I have confusing log during model quantization:
```python
Statistics collection: 100%|██████████| 300/300 [05:13<00:00,  1.05s/it]
Statistics collection: 100%|██████████| 300/300 [06:36<00:00,  1.32s/it]
Applying Fast Bias correction: 100%|██████████| 144/144 [01:52<00:00,  1.28it/s]
````
I see two consecutive statistics collection. 
I suggest adding a progress bar for the first stage algorithms to avoid confusion.

**Review by AlexanderDokuchaev — APPROVED** [review]

**Review by l-bat — APPROVED** [review]

**Review by alexsu52 — APPROVED** [review]

**Comment by alexsu52** [comment]
> It would be great to add progress bar for SQ algorithm, as it is done for BC/FBC. I have confusing log during model quantization:
> 
> ```python
> Statistics collection: 100%|██████████| 300/300 [05:13<00:00,  1.05s/it]
> Statistics collection: 100%|██████████| 300/300 [06:36<00:00,  1.32s/it]
> Applying Fast Bias correction: 100%|██████████| 144/144 [01:52<00:00,  1.28it/s]
> ```
> 
> I see two consecutive statistics collection. I suggest adding a progress bar for the first stage algorithms to avoid confusion.

I think it's better to do it in a separate PR.

**Pull request merged**